.github/uffizzi/uffizzi.production.app-config.yaml

```diff
@@ -0,0 +1,10 @@
+app:
+  title: Backstage Uffizzi Environment
+  baseUrl: ${UFFIZZI_URL}
+
+backend:
+  baseUrl: ${UFFIZZI_URL}
+  cors:
```

Comment:
This shouldn't be needed right? I mean they're on the same url.
It's localhost by default so try making this `cors: null`